### PR TITLE
Add ignore-2.12.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
Please, after merge, add 2.11 to the test matrix (I didn't find it)

https://github.com/ansible-collections/overview/issues/45#issuecomment-813821100

### CODE REVIEW

**Comment by Andersson007** [comment]
dear maintainers, please relaunch this after fixing:
```
ERROR: Found 24 validate-modules issue(s) which need to be resolved:
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'encryption_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'encryption_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> vrfs -> neighbor
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor -> encryption_password
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> vrfs -> neighbor -> encryption_password
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'authentication_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_key
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> message_digest_key
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'keytype' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'keytype' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'message_digest_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> encryption
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> encryption
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> encryption
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> encryption
ERROR: plugins/modules/eos_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set
ERROR: plugins/modules/eos_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set found in aggregate
```
and the other errors. See https://github.com/ansible-collections/overview/issues/45#issuecomment-798891436

**Comment by NilashishC** [comment]
Related to https://github.com/ansible-collections/arista.eos/issues/186.

**Review by GomathiselviS — APPROVED** [review]

**Comment by GomathiselviS** [comment]
@Andersson007 Can you please fix the lint issue?

File: changelogs/fragments/197-add_ignore_txt.yml
Comment (Andersson007) [inline changelogs/fragments/197-add_ignore_txt.yml:1]:
```suggestion
---
trivial:
```

**Comment by Andersson007** [comment]
> @Andersson007 Can you please fix the lint issue?

@GomathiselviS done, thanks

File: changelogs/fragments/197-add_ignore_txt.yml
Comment (Andersson007) [inline changelogs/fragments/197-add_ignore_txt.yml:3]:
```suggestion
  - Add ignore-2.12.txt.
```

**Comment by Andersson007** [comment]
Unrelated errors
```
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'encryption_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'encryption_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> vrfs -> neighbor
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor -> encryption_password
ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> vrfs -> neighbor -> encryption_password
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'authentication_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_key
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> message_digest_key
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'keytype' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'keytype' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'message_digest_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> encryption
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> encryption
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> encryption
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> encryption
ERROR: plugins/modules/eos_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set
ERROR: plugins/modules/eos_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set found in aggregate
```
@GomathiselviS can we merge it forcibly? I usually fix such things if there are 2-3 modules affected. But if i do it in such cases, I think I'll stuck with ~80 collections for several weeks:)

**Comment by GomathiselviS** [comment]
> Unrelated errors
> 
> ```
> ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'encryption_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor
> ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'encryption_password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> vrfs -> neighbor
> ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> neighbor -> encryption_password
> ERROR: plugins/modules/eos_bgp_global.py:0:0: no-log-needed: Argument 'password' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> vrfs -> neighbor -> encryption_password
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'authentication_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_key
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> message_digest_key
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'keytype' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'keytype' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'message_digest_key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> authentication_v3
> ERROR: plugins/modules/eos_ospf_interfaces.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> address_family -> encryption_v3
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> encryption
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'key' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> encryption
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> authentication
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> address_family -> areas -> encryption
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> authentication
> ERROR: plugins/modules/eos_ospfv3.py:0:0: no-log-needed: Argument 'passphrase' in argument_spec could be a secret, though doesn't have `no_log` set found in config -> processes -> areas -> encryption
> ERROR: plugins/modules/eos_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set
> ERROR: plugins/modules/eos_user.py:0:0: no-log-needed: Argument 'sshkey' in argument_spec could be a secret, though doesn't have `no_log` set found in aggregate
> ```
> 
> @GomathiselviS can we merge it forcibly? I usually fix such things if there are 2-3 modules affected. But if i do it in such cases, I think I'll stuck with ~80 collections for several weeks:)

@Andersson007  Can you please include the following in your ignore-2.12.txt.
`plugins/modules/eos_bgp_global.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
plugins/modules/eos_ospf_interfaces.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
plugins/modules/eos_ospfv3.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
plugins/modules/eos_user.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
`

I am working on removing these lines permanently from ignore txt. Once my PR merged, I will remove these lines from this file.

Thank you for working on this.

File: tests/sanity/ignore-2.12.txt
Comment (Andersson007) [inline tests/sanity/ignore-2.12.txt:11]:
```suggestion
plugins/module_utils/network/eos/config/ospfv3/ospfv3.py import-2.6!skip
plugins/modules/eos_bgp_global.py validate-modules:no-log-needed # temporary workaround for unspecified no-log plugins/modules/eos_ospf_interfaces.py validate-modules:no-log-needed # temporary workaround for unspecified no-log plugins/modules/eos_ospfv3.py validate-modules:no-log-needed # temporary workaround for unspecified no-log plugins/modules/eos_user.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
```

File: tests/sanity/ignore-2.12.txt
Comment (Andersson007) [inline tests/sanity/ignore-2.12.txt:12]:
```suggestion
plugins/modules/eos_bgp_global.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
plugins/modules/eos_ospf_interfaces.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
plugins/modules/eos_ospfv3.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
plugins/modules/eos_user.py validate-modules:no-log-needed # temporary workaround for unspecified no-log
```

**Comment by Andersson007** [comment]
@GomathiselviS done, thanks!

**Comment by Andersson007** [comment]
@GomathiselviS can we merge this now?

**Comment by Andersson007** [comment]
I saw you tried to put "gate", so I'll do it, thanks!

**Pull request merged**
